examples/fourier_trafo.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Simple example on the usage of the Fourier Transform."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# Internal
+import odl
+
+
+# Create a uniformly discretized space of functions on the rectangle
+# [-1, 1] x [-1, 1] with 512 samples per dimension. Using a complex data
+# type so one naturally gets the full Fourier transform (see below for real
+# data).
+discr_space = odl.uniform_discr([-1, -1], [1, 1], (512, 512), dtype='complex')
```

Comment:
I actually prefer tuples for the shape, since that's what you get back if you ask for it.